src/views/Accessibility/TestingTemplates/index.tsx

```diff
@@ -0,0 +1,198 @@
+    </div>
+  );
+
+  const downloadVPAT = (
+    <div className="accessibility-testing-templates__downloadBox">
+      <h3 className="margin-top-0">
+        {t('testingTemplates.vpatSection.subSection.downloadVPAT.heading')}
+      </h3>
+      <p>
+        <Link
+          href="https://www.itic.org/policy/accessibility/vpat"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {t(
+            'testingTemplates.vpatSection.subSection.downloadVPAT.line1.linkText'
+          )}
+        </Link>
+        {` `}
+        {t(
+          'testingTemplates.vpatSection.subSection.downloadVPAT.line1.otherText'
+        )}
+      </p>
+      <p>
+        <i className="fa fa-exclamation-circle fa-2x" />
+        {` `}
+        {t('testingTemplates.vpatSection.subSection.downloadVPAT.line2.text')}
+      </p>
+      <p>
+        <Link
+          href="https://www.youtube.com/watch?v=kAkSV9xiJ1A"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {t(
+            'testingTemplates.vpatSection.subSection.downloadVPAT.line3.linkText'
+          )}
+        </Link>
+        {` `}
+        {t(
+          'testingTemplates.vpatSection.subSection.downloadVPAT.line3.otherText'
+        )}
+      </p>
+    </div>
+  );
+
+  const vpatSection = (
+    <div>
+      <h2 id={`testingTemplates_${t('testingTemplates.vpatSection.heading')}`}>
+        {t('testingTemplates.vpatSection.heading')}
+      </h2>
+      <p>{t('testingTemplates.vpatSection.description')}</p>
+      <h3>{t('testingTemplates.vpatSection.subSection.heading')}</h3>
+      <ul className="accessibility-testing-templates__vpat-list">
+        <li>{t('testingTemplates.vpatSection.subSection.item1.text')}</li>
+        {vpatConformanceLevels.map(level => (
+          <p key={level.name}>
+            <span className="text-bold">{level.name}</span> {level.description}
+          </p>
+        ))}
+        <li>{t('testingTemplates.vpatSection.subSection.item2.text')}</li>
+      </ul>
+      {downloadVPAT}
+    </div>
+  );
+
+  return (
+    <div className="grid-container accessibility-testing-templates">
+      <BreadcrumbNav className="margin-y-2">
+        <li>
+          <Link href="/">Home</Link>
+          <i className="fa fa-angle-right margin-x-05" aria-hidden />
+        </li>
+        <li>Templates for 508 testing</li>
+      </BreadcrumbNav>
+      <div className="grid-row grid-gap-lg margin-top-6">
+        <div className="grid-col-9 line-height-body-4">
+          <div className="tablet:grid-col-10">
+            <PageHeading className="margin-top-0">
+              {t('testingTemplates.heading')}
+            </PageHeading>
+            {tableOfContents}
+            {vpatSection}
+            {testPlanSection}
+            {remediationPlanSection}
+          </div>
+        </div>
+        <div className="grid-col-3 accessibility-testing-templates__sidebar">
+          <div>
+            <h4 className="text-bold">
```

Comment:
I think the `className="text-bold"` is extraneous. Headings should be bold by default.